### ISSUE TITLE
'help' attribute for t.Dat gives bootstrap error

### ISSUE THREAD

**Issue by jonaswindey** [gcanti/tcomb-form#164]
When I set a help option to a date field, I get a bootstrap error:
```
Uncaught TypeError: Cannot read property 'id' of undefined.

function getHelp(_ref2) {
  var help = _ref2.help;
  var attrs = _ref2.attrs;

  if (!help) {
    return;
  }
  return _uvdomBootstrap2['default'].getHelpBlock({
    help: help,
    id: attrs.id + '-tip'
  });
}
```
For numbers and strings, there is no problem. 
Using v0.6.1 and react-bootstrap 0.23.5

**Comment by gcanti** [comment]
Thanks @jonaswindey, I'll fix that asap

**Comment by gcanti** [comment]
Just published a fix (v0.6.3)

**Comment by jonaswindey** [comment]
Thanks. works perfectly.
